### ISSUE TITLE
Upload files

### ISSUE THREAD

**Issue by empezar** [Classic-Fortress/classic-fortress.github.io#35]
It would be good to store files on our own server so old configs etc don't get lost.

**Comment by lindstrm** [comment]
A link to paster?

**Comment by empezar** [comment]
No, storing them inside our own database would be a lot better, if someone would take over the hosting for example.
